apps/core/scenes/search/PlacePicker/PlacePickerContent.js

```diff
@@ -31,11 +32,16 @@ type Props = {|
   +travelFrom: ?Location,
   +travelTo: ?Location,
   +modalType: 'DESTINATION' | 'ORIGIN' | 'HIDDEN',
-  +setTravelFrom: (?Location) => void,
-  +setTravelTo: (?Location) => void,
+  +clearLocation: (type: 'travelTo' | 'travelFrom') => void,
+  +setLocation: (type: LocationSearchType, location: Location) => void,
+  +addLocation: (type: LocationSearchType, location: Location) => void,
   +setModalType: ('HIDDEN') => void,
 |};
 
+type State = {
+  text: string,
```

Comment:
`text` is very generic. Could you use a more descriptive name?

Reply:
sure thanks I will try to find better name

Reply:
done `searchText`